Standard/src/Synthesis/MatrixUtils.cs

```diff
@@ -0,0 +1,68 @@
+using System.Diagnostics;
+using System.Numerics;
+using Microsoft.Quantum.Simulation.Core;
+
+namespace Microsoft.Quantum.Synthesis
+{
+    internal class MatrixUtils
+    {
+        // Checks whether given matrix is unitary.
+        public static bool isMatrixUnitary(Complex[,] matrix)
```

Comment:
```suggestion
        public static bool IsMatrixUnitary(Complex[,] matrix)
```

C# language style uses capital method names.